src/psyclone/transformations.py

```diff
@@ -510,11 +513,20 @@ def _directive(self, children, collapse=None):
         :type children: list of :py:class:`psyclone.psyir.nodes.Node`
         :param int collapse: number of nested loops to collapse or None if \
                              no collapse attribute is required.
+        :param bool independent: whether to add the "independent" clause to \
```

Comment:
You've got carried away here - these aren't actually arguments to this method!

Reply:
Ah yes, good point! See ef245a79bbf0517a37fd92c07432c2eb5bcf6b11.